dotcom-rendering/src/web/components/Branding.importable.tsx

```diff
@@ -1,46 +1,132 @@
 import { css } from '@emotion/react';
-import { neutral, textSans } from '@guardian/source-foundations';
+import { ArticleDesign } from '@guardian/libs';
+import { neutral, textSans, until } from '@guardian/source-foundations';
 import { trackSponsorLogoLinkClick } from '../browser/ga/ga';
 
 const brandingStyle = css`
 	padding-bottom: 10px;
 `;
 
-const brandingLabelStyle = css`
-	${textSans.xxsmall()};
-	color: ${neutral[20]};
-`;
+// for liveblog smaller breakpoints article meta is located in the same
+// container as standfirst and needs the same styling as standfirst
+function brandingLabelStyle(palette: Palette, format: ArticleFormat) {
+	const invariantStyles = css`
+		${textSans.xxsmall()}
+	`;
+
+	switch (format.design) {
+		case ArticleDesign.LiveBlog: {
+			return [
+				invariantStyles,
+				css`
+					color: ${neutral[20]};
+
+					${until.desktop} {
+						color: ${palette.text.standfirst};
+					}
+
+					a {
+						color: ${neutral[20]};
+
+						${until.desktop} {
+							color: ${palette.text.standfirst};
+						}
+					}
+				`,
+			];
+		}
+		default: {
+			return [
+				invariantStyles,
+				css`
+					color: ${neutral[20]};
+
+					a {
+						color: ${neutral[20]};
+					}
+				`,
+			];
+		}
+	}
+}
 
 const brandingLogoStyle = css`
 	padding: 10px 0;
 
 	display: block;
-	img {
+`;
+
+// for liveblog smaller breakpoints article meta is located in the same
+// container as standfirst and needs the same styling as standfirst
+const brandingAboutLink = (palette: Palette, format: ArticleFormat) => {
+	const invariantStyles = css`
+		${textSans.xxsmall()}
 		display: block;
+		text-decoration: none;
+		&:hover {
+			text-decoration: underline;
+		}
+	`;
+
+	switch (format.design) {
+		case ArticleDesign.LiveBlog: {
+			return [
+				invariantStyles,
+				css`
+					color: ${palette.text.branding};
+					${until.desktop} {
+						color: ${palette.text.standfirst};
+					}
+					a {
+						color: ${palette.text.branding};
+						${until.desktop} {
+							color: ${palette.text.standfirst};
+						}
+					}
+				`,
+			];
+		}
+		default: {
+			return [
+				invariantStyles,
+				css`
+					color: ${palette.text.branding};
+					a {
+						color: ${palette.text.branding};
+					}
+				`,
+			];
+		}
+	}
+};
+
+const hiddenUntilDesktop = css`
```

Comment:
I thought this was the simplest way to toggle different img sources according to breakpoints, but is this a DCR-approved pattern? Could toggle background-url instead, or potentially picture srcsets? (though neither feel super semantic for this effect..?)

Reply:
There's a `Hide` component, I believe?